src/universal-dapp.js

```diff
@@ -534,10 +510,6 @@ UniversalDApp.prototype.deployLibrary = function(contractName, cb) {
     }
 };
 
-UniversalDApp.prototype.clickNewContract = function ( self, $contract, contract ) {
```

Comment:
No idea about that one. According to *blame* it is there since the beginning: https://github.com/ethereum/browser-solidity/commit/bf0acf274b4726902f8e47efc429bae56d78595d